sdk/ml/azure-ai-ml/azure/ai/ml/operations/_evaluator_operations.py

```diff
@@ -92,7 +94,7 @@ def __init__(
 
     @monitor_with_activity(ops_logger, "Evaluator.CreateOrUpdate", ActivityType.PUBLICAPI)
     def create_or_update(  # type: ignore
-        self, model: Union[Model, WorkspaceAssetReference]
+        self, model: Union[Model, WorkspaceAssetReference], **kwargs: Any
```

Comment:
In case you hit linting issue, use this

**kwargs: Any,  # pylint: disable=unused-argument

Reply:
It looks like it works in the new pyling, however it does not like docstring unless it is in the class definition.